RecoPPS/Local/python/ctppsDiamondLocalReconstruction_cff.py

```diff
@@ -6,6 +6,11 @@
 # local track fitting
 from RecoPPS.Local.ctppsDiamondLocalTracks_cfi import ctppsDiamondLocalTracks
 
+from CalibPPS.ESProducers.ppsTimingCalibrationLUTESSource_cfi import ppsTimingCalibrationLUTESSource
+
+ppsTimingCalibrationLUTESSource.calibrationFile = cms.FileInPath('RecoPPS/Local/data/LUT_cal_test.json')
+ctppsDiamondRecHits.timingCalibrationLUTTag=cms.string('ppsTimingCalibrationLUTESSource:')
+
```

Comment:
Hi @ChrisMisan I understand that you are reading the json since 12_3_0_pre3 is not out yet and you can not create a candidate GT with the needed tag, which I this is this one: [PPSDiamondTimingCalibrationLUT_test](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/tags/PPSDiamondTimingCalibrationLUT_test) (already available in CondDB).
There is however a method to customize the GT to read a specific tag from the DB:
```
process.GlobalTag.globaltag = 'auto:run3_data' # make sure you chose the right one here, this is just an example
process.GlobalTag.toGet = cms.VPSet()
process.GlobalTag.toGet.append(
  cms.PSet(record = cms.string(“PPSTimingCalibrationLUTRcd”),
           tag =  cms.string(“PPSDiamondTimingCalibrationLUT_test”),
           connect = cms.untracked.string("frontier://FrontierProd/CMS_CONDITIONS")
)
```
You can find more details in [slides 17-18](https://indico.cern.ch/event/1026459/contributions/4332410/attachments/2233312/3789053/talk1%20-%20AlCa_DB_tutorial_introduction_2021.pdf#page=17&zoom=auto,-232,600) of the AlCaDB tutorial from last year.

This method is usually not recommended by AlCa (we prefer to have the needed tags included in the GT),
but for the moment I think this is your best option.